web/app/css/sidebar.css

```diff
@@ -129,6 +169,25 @@ ul.ant-menu.ant-menu-sub {
   background: var(--sider-black);
 }
 
+/* override ant form select in sidebar */
+.ant-form-inline .ant-form-item {
+  margin-right: 0;
+}
+
+.sidebar .sidebar-menu-item .ant-select-selection {
+  background-color: rgba(255, 255, 255, 0.8);
+  border: none;
+  line-height: var(--spacer-quintuple);
+}
+
+.ant-select-selection--single {
+  height: auto;
+}
+
+.ant-select-selection__rendered {
+  line-height: var(--spacer-quintuple);
+}
+
```

Comment:
Looks like the sidebar may need additional style specification when it is in its collapsed state. This can be triggered by hitting the sidebar footer button or shrinking the page's width so that the sidebars breakpoints are triggered.